add-domain

```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
+newappmail.discoverynevers.com
+app-uniswap.org-vote-eth-swap.site
```

Comment:
Seems dead to me from both manual lookup and @pyfunceble

![image](https://user-images.githubusercontent.com/44526987/126140881-7db9491e-8d72-48e2-a1f7-6fd0e9b227cd.png)